opaque_keys/edx/locator.py

```diff
@@ -1425,6 +1394,7 @@ def __init__(
         """
         Instantiate a new BundleDefinitionLocator
         """
+        warnings.warn("BundleDefinitionLocator and Blockstore are deprecated!" DeprecationWarning, stacklevel=2)
```

Comment:
```suggestion
        warnings.warn("BundleDefinitionLocator and Blockstore are deprecated!", DeprecationWarning, stacklevel=2)
```

I think it's missing a comma between args?